src-electron/validation/validation.js

```diff
@@ -77,15 +96,36 @@ function validateSpecificAttribute(endpointAttribute, attribute) {
       if (!checkAttributeBoundsFloat(attribute, endpointAttribute))
         defaultAttributeIssues.push('Out of range')
     } else if (types.isSignedInteger(attribute.type)) {
-      if (!isValidSignedNumberString(endpointAttribute.defaultValue))
+      if (!isValidSignedNumberString(endpointAttribute.defaultValue)) {
         defaultAttributeIssues.push('Invalid Integer')
-      if (!checkAttributeBoundsInteger(attribute, endpointAttribute))
+      } else if (
+        //we shouldn't check boundaries for an invalid number string
+        !(await checkAttributeBoundsInteger(
+          attribute,
+          endpointAttribute,
+          db,
+          zapSessionId,
+          clusterRef
+        ))
+      ) {
         defaultAttributeIssues.push('Out of range')
+      }
     } else {
-      if (!isValidNumberString(endpointAttribute.defaultValue))
+      // TODO this case does the same thing as the one with signed integer
```

Comment:
Remove todo messages

Reply:
i will , however this is not connected to this task and as i could see there are many todos like this are left in the project

this and another function here is in need of a rework , i can open a separate issue for it if you wish

Reply:
To do's in the code are difficult to track. Log issues for these and remove your todos from here.

Reply:
will do